### PR TITLE
Issue #1055 Invalid Json fetched from jv_snapshot.state column oracle…

### PR DESCRIPTION
**Pull request by WojtekPWas**
… sql database.
Two small changes and one test class.
I could not reproduce the problem.
Test returns correct data also without my modification.
This is probably because of database maintenance

### CODE REVIEW

**Review by bartoszwalacik — CHANGES_REQUESTED** [review]
needs work, see my comments in https://github.com/javers/javers/issues/1055

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java [hunk 1]
```diff
@@ -9,24 +9,53 @@
...
 import java.sql.ResultSet;
 import java.sql.SQLException;
 import java.util.List;
 import java.util.Map;
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import static org.javers.repository.sql.schema.FixedSchemaFactory.*;
-import static org.javers.repository.sql.session.Parameter.*;
+import static org.javers.repository.sql.schema.FixedSchemaFactory.COMMIT_AUTHOR;
```
Comment (bartoszwalacik) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java:23]: no :)

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java [hunk 2]
```diff
@@ -9,24 +9,53 @@
...
+import static org.javers.repository.sql.schema.FixedSchemaFactory.SNAPSHOT_STATE;
+import static org.javers.repository.sql.schema.FixedSchemaFactory.SNAPSHOT_TYPE;
+import static org.javers.repository.sql.schema.FixedSchemaFactory.SNAPSHOT_VERSION;
+import static org.javers.repository.sql.session.Parameter.bigDecimalParam;
+import static org.javers.repository.sql.session.Parameter.instantParam;
+import static org.javers.repository.sql.session.Parameter.localDateTimeParam;
+import static org.javers.repository.sql.session.Parameter.longParam;
+import static org.javers.repository.sql.session.Parameter.stringParam;
 
 class SnapshotQuery {
```
Comment (bartoszwalacik) [inline javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java:50]: If Dialect is needed in SnapshotQuery, maybe you can get it from selectBuilder.session.dialect?

**Review by bartoszwalacik — CHANGES_REQUESTED** [review]

File: javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java (hunk 2 above)
Comment (WojtekPWas) [reply javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java]: I have pushed a new version. I have used session.dialect, but the classes Session, Dialect and Dialects are more public now and it is of course a drawback. I can not say whether it is better comparing with the previous version with additional injections.

Comment (bartoszwalacik) [reply javers-persistence-sql/src/main/java/org/javers/repository/sql/finders/SnapshotQuery.java]: ok, I will take a look

**Pull request merged**